test/Test/Toml/Gen.hs

```diff
@@ -307,8 +314,10 @@ genText = genNotEscape $ fmap Text.concat $ Gen.list (Range.constant 0 256) $ Ge
     , genPunctuation
     , genUniHex4Color
     , genUniHex8Color
+    --, genUnicodeChar
```

Comment:
Should this be uncommented back?

Reply:
``genUnicodeChar`` is the function that I wrote myself thinking that I might be helpful to add it into the testing phase. But if I enable it, the test will fail. 
The reason why it fails is mentioned above but I can repeat it. Character ``č`` is transformed into ``\U0000010d`` but decoding it back will give ``\U0000010d`` again, not ``č`` so the ``encode . decode`` tests will fail. 
Currently, tests are only include escaped unicode characters in ``encode . decode`` phase.